Product/Production/Services/AuditRepositoryCore/src/main/java/gov/hhs/fha/nhinc/audit/AuditLogger.java

```diff
@@ -92,14 +94,14 @@ public void auditResponseMessage(T request, K response, AssertionType assertion,
         String serviceName) {
 
         LOG.trace("--- Before auditing of response message ---");
-        if (getAuditLogger() != null) {
+        if (getAuditLogger() != null && isAuditLoggingOn(serviceName)) {
```

Comment:
Should these cases be reverse? (i.e., don't instantiate the logger unless audit logging is on)